.github/workflows/stylelint-pr.yml

```diff
@@ -0,0 +1,47 @@
+name: Stylelint PR Diffs
```

Comment:
suggestion: Lint scss styles 

Something just a bit more plain english readable. Just a suggestion

Reply:
As discussed in slack. Going leave this as is.

Ideally we should call it this - but that can be used for when we have all lint errors down and we are running it all the time and not just on the PR diff set